FirebaseMLModelDownloader/Tests/Unit/ModelDownloaderUnitTests.swift

```diff
@@ -123,6 +123,13 @@ final class ModelDownloaderUnitTests: XCTestCase {
     }
   }
 
+  /// Test on-device logging.
+  func testDeviceLogging() {
+    DeviceLogger.logEvent(level: .error,
+                          message: "This is a test error logged to device.",
+                          messageCode: .testError)
+  }
```

Comment:
No. For now all this test does is log a "error" because nothing in the SDK so far is dire enough to warrant that level. I wasn't sure how to check if something was logged to device. Is there a way?